src/components/Products/AllProducts.js

```diff
@@ -19,50 +19,41 @@ class AllProducts extends Component {
 
       var ChangeHidden = (event) => {
```

Comment:
Uncertain if this is required.
We show and hide the product name and price based on hover/focus